version.json

```diff
@@ -1,7 +1,6 @@
 {
   "version": "15.1-preview",
   "publicReleaseRefSpec": [
-    "^refs/heads/master$",
-    "^refs/heads/xplat$"
+    "^refs/heads/.*-rtw"
```

Comment:
I agree.  I really wanted to go off tags instead but we don't tag stuff until after we RTM.  I think we'll just need to come up with a naming scheme and add to this as we go.  Although update 1 might be `vs15-update1-rtw` which would match this pattern...